### PR TITLE
Update semver docs and mention default behavior

### PR DESCRIPTION
**Pull request by ajp-io**

### CODE REVIEW

**Review by sgalsaleh — DISMISSED** [review]

**Comment by ajp-io** [comment]
@replicatedhq/replicated-docs Can you review these updates to our docs on semantic versioning support?

File: content/vendor/packaging/promoting-releases.md [hunk 1]
```diff
@@ -35,11 +35,18 @@ Additionally, each promoted release must be given a required status (required or
...
-In order for this label to be used to sequence releases on a channel, semantic versioning must be enabled by turning on the `Require semver for releases promoted to this channel` toggle in the channel settings.
-If you use semantic versioning on a channel, we recommend that you always use semantic versioning on that channel.
-This toggle is disabled by default.
+### Enabling Semantic Versioning
+
+Semantic versioning must be enabled on a per-channel basis. It can be enabled for existing channels by enabling **Enable Semantic Versioning** in the channel settings in the vendor portal. When creating a new channel in the vendor portal, select **Prevent promoting releases with an invalid semantic version to this channel** to enable this option. Semantic versioning can also be enabled and disabled for an existing channel with the [`kots channel enable-semantic-versioning`](/vendor/cli/channel-enable-semantic-versioning) and [`kots channel disable-semantic-versioning`](/vendor/cli/channel-disable-semantic-versioning) commands, respectively.
```
Comment (ajp-io) [inline content/vendor/packaging/promoting-releases.md:40]: Do you think the second sentence here has the word enabled/enabling too many times in one sentence now?

Comment (jonquil2002) [reply content/vendor/packaging/promoting-releases.md:40]: Argh, good call. Need more coffee. I'll do another edit.

**Review by jonquil2002 — APPROVED** [review]
Added a second commit to reduce the use of "enable".

**Pull request merged**